<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogNotebookClass</class>
 <widget class="QMainWindow" name="LogNotebookClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Notebook</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>376</width>
      <height>218</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="exportExcel"/>
    <addaction name="exportCSV"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="resetData"/>
    <addaction name="removeName"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="manageCloud"/>
    <addaction name="autoFillTask"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exportSheet">
   <property name="text">
    <string>Export Time Sheet</string>
   </property>
  </action>
  <action name="exportWorked">
   <property name="text">
    <string>Export Time Worked</string>
   </property>
  </action>
  <action name="exportSheet_2">
   <property name="text">
    <string>Log...</string>
   </property>
  </action>
  <action name="exportWorked_2">
   <property name="text">
    <string>Volunteer Hours...</string>
   </property>
  </action>
  <action name="clearLog">
   <property name="text">
    <string>Reset Log...</string>
   </property>
  </action>
  <action name="resetData">
   <property name="text">
    <string>Reset Data...</string>
   </property>
  </action>
  <action name="openTimeSheet">
   <property name="text">
    <string>Log...</string>
   </property>
  </action>
  <action name="openTimeWorked">
   <property name="text">
    <string>Volunteer Hours...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="exportExcel">
   <property name="text">
    <string>Export To Excel...</string>
   </property>
  </action>
  <action name="exportCSV">
   <property name="text">
    <string>Export To CSV...</string>
   </property>
  </action>
  <action name="manageCloud">
   <property name="text">
    <string>Manage Cloud Backup</string>
   </property>
  </action>
  <action name="backup">
   <property name="text">
    <string>Back Up Now</string>
   </property>
  </action>
  <action name="getLastBacup">
   <property name="text">
    <string>Get Last Backup</string>
   </property>
  </action>
  <action name="setBackupFrequency">
   <property name="text">
    <string>Set Backup Frequency</string>
   </property>
  </action>
  <action name="removeName">
   <property name="text">
    <string>Edit Names...</string>
   </property>
  </action>
  <action name="autoFillTask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Fill Task</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LogNotebook.qrc"/>
 </resources>
 <connections/>
</ui>
